sdk/README.md

```diff
@@ -87,7 +87,76 @@ After that you can start the application by:
 node index.js
 ```
 
-More usage examples and configuration samples will be made available during the alpha phase on the official Lisk [documentation portal](http://docs.lisk.io).
+### Configure your blockchain parameters
+
+You can also define your blockchain application parameters such as `block time`, `epoch time`, `max transactions per block` and more with an optional configurations object.
```

Comment:
`block time` -> Prefer to have consistent naming like `BLOCK_TIME` for those properties?